src/libs/utils.ts

```diff
@@ -135,14 +133,6 @@ export const snapshotReferenceMissingError = (snapshotReferenceName) => {
   return `The requested snapshot reference '${snapshotReferenceName}' could not be found in this workspace.`;
 };
 
-// Returns a message explaining why the user can't edit the workspace, or undefined if they can
-export const editWorkspaceError = ({ accessLevel, workspace: { isLocked } }) => {
-  return cond(
-    [!canWrite(accessLevel), () => 'You do not have permission to modify this workspace.'],
-    [isLocked, () => 'This workspace is locked.']
-  );
-};
-
 // Returns a message explaining why the user can't compute in the workspace, or undefined if they can
 export const computeWorkspaceError = ({ canCompute, workspace: { isLocked } }) => {
```

Comment:
To be consistent, should this be moved to workspace-utils as well?

Reply:
Since that isn't in the scope of this PR's objective (decoupling the two files), I'm not going to commit to doing it here, but I'm gaining a lot of confidence that there is an upcoming PR to address some additional opportunities.